### ISSUE TITLE
incorrect to_query

### ISSUE THREAD

**Issue by kostya** [werner/crystal_on_steroids#9]
```
crystal eval 'require "crystal_on_steroids"; h = {data: "⬤"}.to_query; p h'
```

```
"%7Bdata%3A%20%22%E2%AC%A4%22%7D"
```
```
ruby -e 'require "active_support/all"; h = {data: "⬤"}.to_query; p h'
```
```
"data=%E2%AC%A4"
```

**Comment by kostya** [comment]
```
crystal eval 'require "crystal_on_steroids"; h = {"x" => {"y" => "z"}}; p h.to_query'
```
```
Error in line 1: instantiating 'Hash(String, Hash(String, String))#to_query()'

in lib/crystal_on_steroids/src/crystal_on_steroids/to_query.cr:78: instantiating 'to_query(Nil)'

  def to_query(namespace = nil)
  ^

in lib/crystal_on_steroids/src/crystal_on_steroids/to_query.cr:80: instantiating 'Hash(String, Hash(String, String))#each()'

      compact.each do |key, value|
              ^~~~

in lib/crystal_on_steroids/src/crystal_on_steroids/to_query.cr:80: instantiating 'Hash(String, Hash(String, String))#each()'

      compact.each do |key, value|
              ^~~~

in lib/crystal_on_steroids/src/crystal_on_steroids/to_query.cr:82: instantiating 'HTTP::Params::Builder#add(String, Hash(String, String))'

          form.add (namespace ? "#{namespace}[#{key.to_s}]" : key.to_s), value
               ^~~

in /usr/local/Cellar/crystal-lang/0.24.2_1/src/http/params.cr:336: no overload matches 'HTTP::Params.encode_www_form_component' with types Hash(String, String), IO+
Overloads are:
 - HTTP::Params.encode_www_form_component(string : String, io : IO)

        Params.encode_www_form_component value, @io if value
               ^~~~~~~~~~~~~~~~~~~~~~~~~
```

```
ruby -e 'require "active_support/all"; h = {"x" => {"y" => "z"}}; p h.to_query'
```

```
"x%5By%5D=z"
```

**Comment by werner** [comment]
Fix in af8b6a6, thanks for reporting the bug.

Best regards.
